app/main.py

```diff
@@ -1,6 +1,22 @@
+import json
+import os
+from dotenv import load_dotenv
+
+import requests
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json?"
+API_KEY = os.environ["API_KEY"]
```

Comment:
It's important to check if the `API_KEY` environment variable is set before using it. Consider adding a check to ensure the key is present and handle the case where it might be missing.